app/src/main/java/com/alphawallet/app/service/TokensService.java

```diff
@@ -956,7 +950,7 @@ private void blankFiltersForZeroBalance(boolean mainNetActive)
             if (!networkFilter.contains(lockedChain)) networkFilter.add(lockedChain);
         }
 
-        if (networkFilter.size() == 0) networkFilter.add(ethereumNetworkRepository.getDefaultNetwork(mainNetActive));
+        if (networkFilter.size() == 0) networkFilter.add(ethereumNetworkRepository.getDefaultNetwork(true));
 
```

Comment:
This can have the boolean removed - it's now always CustomViewSettings.primaryChain